ovs-persistence/src/main/java/org/dcsa/ovs/persistence/entity/TransportCall.java

```diff
@@ -14,7 +14,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@Setter(AccessLevel.PRIVATE)
+@Setter(AccessLevel.PUBLIC)
```

Comment:
Is this change needed?

Reply:
Had to set vessel explicitly to Transport call.